web/src/containers/Order/index.tsx

```diff
@@ -297,6 +379,13 @@ class OrderInsert extends React.PureComponent<Props, StoreProps> {
         this.props.setCurrentPrice(0);
     };
 
+    private listenInputTrigger = () => {
+        this.setState({
+            trigger: undefined,
```

Comment:
better assign null, because `undefined` is undefined undefined type, but 'null' is defined undefined type